modules/apps/web-experience/iframe/iframe-web/src/main/java/com/liferay/iframe/web/configuration/IFramePortletInstanceConfigurationModelAction.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.iframe.web.configuration;
+
+import com.liferay.configuration.admin.action.ConfigurationModelAction;
+import com.liferay.configuration.admin.action.ConfigurationModelActionException;
+
+import java.util.Dictionary;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Drew Brokke
+ */
+@Component(
+	immediate = true,
+	property = {
+		"model.class.name=com.liferay.iframe.web.configuration.IFramePortletInstanceConfiguration"
+	},
+	service = ConfigurationModelAction.class
+)
+public class IFramePortletInstanceConfigurationModelAction
```

Comment:
You probably just added this commit temporarily, but just in case, this should go into the documentation, not in master.